src/bsp/platform/cc3200/xi_bsp_io_net_cc3200.c

```diff
@@ -24,19 +36,93 @@ xi_bsp_io_net_state_t xi_bsp_io_net_create_socket( xi_bsp_socket_t* xi_socket )
         return XI_BSP_IO_NET_STATE_ERROR;
     }
 
-    //*xi_socket = socket( SL_AF_INET, SOCK_STREAM, 0 );
-    *xi_socket = sl_Socket( AF_INET, SOCK_STREAM, 0 );
+    int retval = 0;
+
+#ifdef XI_BSP_IO_NET_TLS_SOCKET
+
+    long lFileHandle;
+    unsigned long ulToken;
+
+    if ( ( retval = WriteFileToDevice( &ulToken, &lFileHandle ) ) < 0 )
+    {
+        printf( "ERROR: WriteFileToDevice result: %d\n", retval );
```

Comment:
xi_debug_output is in xi_debug.h do we want to "teach" the porting developer to use libxively debug solution?